src/Sarif.WorkItemFiling/Extensions.cs

```diff
@@ -16,7 +18,7 @@ public static WorkItemModel CreateWorkItemModel(this SarifLog sarifLog, string p
             {
                 AdditionalData = sarifLog,
                 Area = projectName,
-                Attachment = new WorkItemFiling.Attachment
+                Attachment = new MicrosoftWorkItemFiling.Attachment
```

Comment:
>MicrosoftWorkItemFiling [](start = 33, length = 23)

Nit: We could replace this with Microsoft.WorkItemFiling and then there is no need for the extra using on line 7

Reply:
Yes, of course! I tried using a more minimal qualifier, WorkItemFiling.Attachment but this won't work because 'WorkItemFiling' is a suffix on the current namespace. Your suggestion works thought. :)

---
In reply to: [377356810](https://github.com/microsoft/sarif-sdk/pull/1749#discussion_r377356810) [](ancestors = 377356810)